### ISSUE TITLE
gencli: Repeated enums not supported

### ISSUE THREAD

**Issue by ericwenn** [googleapis/gapic-generator-go#319]
The generated CLI does not support repeated enums, and treats it as non-repeated, breaking the build.

For example:
```proto
enum MyEnum {
  MY_ENUM_UNSPECIFIED  = 0;
  MY_ENUM_FOO = 1;
}

message Request {
  repeated MyEnums my_enums = 1;
}
```

generates:
```go
// should be []string
var RequestMyEnums string
// ...
RequestCmd.Flags().StringSliceVar(&RequestInputMyEnums, "my_enums", []string{}, "")
// ...
// should iterate over `RequestInputMyEnums` and append.
RequestInput.MyEnums = pb.MyEnum(pb.MyEnum_value[strings.ToUpper(RequestInputMyEnums)])
```

**Comment by noahdietz** [comment]
Thanks for using the CLI generator and thanks for the issue. I’ll take a look at this soon.

**Comment by noahdietz** [comment]
Accidentally closed it sorry.

**Comment by noahdietz** [comment]
Released in v0.12.1. Give it a shot please!

**Comment by ericwenn** [comment]
Just tried it without issues. Thanks a lot @noahdietz! 🎆
